Makefile

```diff
@@ -398,6 +398,15 @@ clippy_cuda_backend: install_rs_check_toolchain
 	RUSTFLAGS="$(RUSTFLAGS)" cargo "$(CARGO_RS_CHECK_TOOLCHAIN)" clippy --all-targets \
 		-p tfhe-cuda-backend -- --no-deps -D warnings
 
+.PHONY: check_rust_bindings_change # Check rust bindings are up to date for tfhe-cuda-backend
+check_rust_bindings_change: 
```

Comment:
actually this is `check_rust_bindings_did_not_change` as it's successful if it did not change :)